File: EASY/PRIME-TESTING/pt.java
class Solution{
    public boolean isPrime(int n){
        if (n<=1) return false;
        for (int i= 2; i <= Math.sqrt(n); i++){
            if(n%i == 0) return false;
        }
        return true;
    }
}

public class pt{
    public static void main(String[] args) {
        Solution s1 = new Solution();
        System.out.print("Enter a number: ");
        int n = new java.util.Scanner(System.in).nextInt();
        System.out.println(s1.isPrime(n) ? "Prime" : "Not Prime");
    }

}

File: EASY/PRIME-TESTING/pt.cpp
#include <iostream>
#include <cmath>
using namespace std;

class Solution {
public:
    bool isPrime(int n) {
        if (n <= 1) {
            return false;
        }
        for (int i = 2; i <= sqrt(n); i++) {
            if (n % i == 0) {
                return false;
            }
        }
        return true;
    }
};

int main() {
    Solution s;
    int n;
    cout << "Enter a number: ";
    cin >> n;

    if (s.isPrime(n)) {
        cout << "Yes" << endl;
    } else {
        cout << "No" << endl;
    }

    return 0;
}